src/dxtbx/nexus/nxmx.py

```diff
@@ -872,7 +872,10 @@ def saturation_value(self) -> int | None:
         to the underload_value.
         """
         if "saturation_value" in self._handle:
-            return int(self._handle["saturation_value"][()])
+            try:
+                return int(self._handle["saturation_value"][()])
+            except TypeError as e:
+                logger.warning(f"Error extracting {self.path}: {e} ")
```

Comment:
Does `None` say what we want? Otherwise I think this is very sensible.

Reply:
If `saturation_value` is `None` then it will default to `0x7FFFFFFF`, which is probably relatively sensible in the absence of any better knowledge?
https://github.com/dials/dxtbx/blob/9339a9e1cfd50e52c5996dce1ac65034100a0c4d/src/dxtbx/nexus/__init__.py#L274-L278